### DISCUSSION TITLE
Forward Node Use

### DISCUSSION THREAD

**Discussion by stevefxp1** [Security-Onion-Solutions/securityonion#5454] [Unsupported Versions] [answered]
Hello all,

I am new to Security Onion and am trying to understand some of the components that make up the full stack. Is a forward node a node that sniffs traffic via a tap or port mirror and then "forwards" that information downstream? I am testing an all in one PC with all components sitting on it but am starting to consider how to split off some pieces.

Thanks,
Steve

**Comment by ranguli** [comment, marked as answer]
As someone who is also new to Security Onion, I've found the online [documentation](https://docs.securityonion.net/en/2.3/index.html) to be indispensable for wrapping your head around how it works. In particular there's a section [here](https://docs.securityonion.net/en/2.3/architecture.html#forward-node) that covers the different types of nodes and their purpose which should hopefully answer your question. A forward node is described as the following:

> **Forward Node**
A forward node is a sensor that forwards all logs via Filebeat to Logstash on the manager node, where they are stored in Elasticsearch on the manager node or a search node (if the manager node has been configured to use a search node). From there, the data can be queried through the use of cross-cluster search.

Hope that helps!
Joshua
